sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -99,6 +99,19 @@ def __init__(self, loss, penalty='l2', alpha=0.0001, C=1.0,
         self._validate_params()
 
     def set_params(self, **kwargs):
+        """
+        Set and validate the parameters of estimator.
```

Comment:
This should be,
```py
"""Set and validate
```
we disabled the rule numpy that would have produced this output.